bin/validate.py

```diff
@@ -7,7 +7,8 @@
 import zipfile
 
 if __name__ == "__main__":
-    predictions_path = sys.argv[1]
+    submision_id = sys.argv[1]
```

Comment:
nit: typo in `submission_id`

but does this get called anywhere? I don't see this referenced in the rest of the code, unless I missed it

Reply:
No, it isn't used in the current `validate.py` script (which is just a dummy/placeholder). But since we need the `submission_id` in `dynamic_challenge_validate.py` I added it to the positional arguments for the placeholder so we can still use the same `VALIDATE` process for both workflows for now.